windows/binary_security_descriptor.ksy

```diff
@@ -0,0 +1,244 @@
+meta:
+  id: binarysd
+  endian: le
+  title: Windows Binary Security Descriptor parser
+  license: MIT
+  ks-version: 0.9
+doc: |
+  Creator: Florian Bausch, ERNW Research GmbH, https://ernw-research.de
+doc-ref: |
+  https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/2918391b-75b9-4eeb-83f0-7fdc04a5c6c9
+  https://docs.microsoft.com/en-us/windows/win32/secauthz/sid-strings
+  https://docs.microsoft.com/en-us/windows/win32/secauthz/well-known-sids
+  https://itconnect.uw.edu/wares/msinf/other-help/understanding-sddl-syntax/
+  https://github.com/ernw/quarantine-formats/blob/master/docs/Binary_Security_Descriptor.md
+seq:
+  - id: fixed
+    contents: [0x03, 0, 0, 0, 0x02, 0, 0, 0]
```

Comment:
Can they be 2 `u4`?

Reply:
Might be, but I only saw this fixed sequence of bytes in my samples.